jest.config.ts

```diff
@@ -6,10 +6,16 @@ const config: Config.InitialOptions = {
     escapeString: true,
     printBasicPrototype: true,
   },
-  testMatch: ["**/__tests__/**/*.test.ts"],
+  testMatch: ["**/__tests__/**/*.ts"],
+  testPathIgnorePatterns: [
+    "/node_modules/",
+    "/fixtures/",
+    "/snapshotSerializers/",
+  ],
```

Comment:
We were missing out on actually running a bunch of tests that were not ending on `.test.ts`